src/fst.jl

```diff
@@ -1139,11 +1139,11 @@ function eq_to_in_normalization!(fst::FST, always_for_in::Bool)
             op.val = "="
             op.len = length(op.val)
         end
-    elseif fst.typ === Block || fst.typ === Brackets
+    elseif !is_leaf(fst)
```

Comment:
what's the reasoning for changing this?

Reply:
Without this I couldn't make the `[for ... if ...]` syntax working. Also a content of the original PR:
https://github.com/domluna/JuliaFormatter.jl/pull/380/files#diff-fdbcedac9cc0bafcd2fa355f83f9558522f56e8e432e9a3775f59612dce9fe8aR1029